scripts/alloyBuilder.js

```diff
@@ -208,7 +234,7 @@ program
   .description(
     "Tool for generating custom Adobe Experience Platform Web SDK builds.",
   )
-  .version("1.0.0");
+  .version("1.1.0");
```

Comment:
How is this version used? Is it included in the build? Since this is included as part of the same repo and versioned with Alloy, do we need a version?

Reply:
It is just the script version. You can see it when your run the script with the `-v` flag. I can remove it, but it seems more like a standard script that you see everywhere these days.

Reply:
Alternatively, we could add the alloy version instead.